### PR TITLE
Draft for HTTP response compression blog

### LINKED ISSUE #291: Automatically compress http responses
Things we need to ensure we include in the blog:

    Document order relative to other similar servlet body filters
    Document configuration error conditions, how to verify configuration
    is as expected(types are what is expected for example). There are also multiple examples in the design document.

### PR DESCRIPTION
**Pull request by mrsaldana**
fixes #291

### CODE REVIEW

File: drafts/HTTP-Response-Compression.adoc [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+= Setting HTTP Response Compression with Open Liberty
+Manuel Saldana <https://github.com/mrsaldana>
+:imagesdir: /
+:url-prefix:
+:bl: pass:[ +]
+:star: pass:[*]
+:add: pass:[+]
+
+
+It is now possible to configure Open Liberty so that HTTP responses are considered for compression. By compressing the sizes of HTTP responses, bandwidth is reduced, which in turn decreases exchanges times between HTTP clients and Open Liberty servers. Sounds good so far... but, what does the statement `_HTTP responses are considered for compression_` actually mean?
```
Comment (pnicolucci) [inline drafts/HTTP-Response-Compression.adoc:20]: 'decreases exchanges times` to `decreases exchange times`

Comment (mrsaldana) [reply drafts/HTTP-Response-Compression.adoc]: changed, thanks!

File: drafts/HTTP-Response-Compression.adoc [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+      Accept-Encoding: gzip;q=0, *
+
+      - The HTTP client can specify it does *not* support either the `gzip` or `deflate` algorithms. In this case, no compression algorithm will be chosen.
+
+      Accept-Encoding: gzip;q=0, deflate;q=0, *
+
+
+If no content encoding algorithm was chosen by the steps above, or if the encoding algorithm is set to `identity`, *no* compression will be realized on the response.
+
+Additionally, since the `Accept-Encoding` header influences the Open Liberty's process for selecting and representing the response, a `Vary` header with the value of `Accept-Encoding` is also added to the response. This will happen regardless of whether the response is ultimately compressed or not, as long as the compression feature is configured in the `server.xml`.
```
Comment (pnicolucci) [inline drafts/HTTP-Response-Compression.adoc:113]: Give some information on the vary header we shouldn't assume everyone knows what it is / does.

File: drafts/HTTP-Response-Compression.adoc [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    Accept-Encoding: deflate;q=0.5, gzip;q=1.0
+
+      - The HTTP client can specify multiple encoding algorithms with the highest quality value. In such cases, the server will pick the `gzip` algorithm if able.
+
+      Accept-Encoding: deflate;q=1.0, gzip;q=1.0
+
+      - The HTTP client may also indicate all encodings are supported. Since the `gzip` algorithm is supported, the server will choose it.
+
+      Accept-Encoding: *
+
```
Comment (pnicolucci) [inline drafts/HTTP-Response-Compression.adoc:101]: Lets add a note for these examples that says if preferred also isn't configured

**Pull request merged**